config/packages/monolog.yaml

```diff
@@ -0,0 +1,64 @@
+monolog:
+  channels:
+    - deprecation # Deprecations are logged in the dedicated "deprecation" channel when it exists
+
+when@dev:
+  monolog:
+    handlers:
+      main:
+        type: stream
+        path: "%kernel.logs_dir%/%kernel.environment%.log"
+        # Or use: "debug" instead of "info" for more log (debug) messages
+        level: info
+        channels: ["!event"]
+        # Enable full stacktrace
+        include_stacktraces: true
+      # uncomment to get logging in your browser
+      # you may have to allow bigger header sizes in your Web server configuration
+      #firephp:
+      #    type: firephp
+      #    level: info
+      #chromephp:
+      #    type: chromephp
+      #    level: info
+      console:
+        type: console
+        process_psr_3_messages: false
+        channels: ["!event", "!doctrine", "!console"]
+
+when@test:
+  monolog:
+    handlers:
+      main:
+        type: fingers_crossed
+        action_level: error
+        handler: nested
+        excluded_http_codes: [404, 405]
+        channels: ["!event"]
+      nested:
+        type: stream
+        path: "%kernel.logs_dir%/%kernel.environment%.log"
+        level: debug
+
+when@prod:
+  monolog:
+    handlers:
+      main:
+        type: rotating_file
```

Comment:
I think it's here

Reply:
Yea I tried to enable log rotation but you can't have apparently the nice features of this other type (this finger a crossed) plus log at once. It's a shame. Let me see what we should wanna use in monolog.

Reply:
I think it should be possible to do something like this in order to use both `fingers_crossed` and log rotation

```yaml
monolog:
  handlers:
    main:
      type: fingers_crossed
      action_level: error
      handler: nested
      buffer_size: 50 # How many messages should be saved? Prevent memory leaks
    nested:
      type: rotating_file
      path: "%kernel.logs_dir%/%kernel.environment%.log"
      level: debug
      formatter: monolog.formatter.json
      max_files:
```

currently testing them in my instance, might report back later if it works

Reply:
Ah yes exactly add it to nested.